src/scss/use-cases/_general.scss

```diff
@@ -27,18 +27,32 @@
 
 /// Subscript text
 @mixin oTypographySub {
-	@include oTypographySans(-2);
-	display: inline-block;
-	margin-bottom: _oTypographyAdjustUnit(-5px);
+	// make subscript smaller and sans-serif
```

Comment:
lots of shared styles between these too, do you think they'd always be the same except for the margin offset direction? worth a private `_oTypographySmolBoi` mixin?

Reply:
Hmm maybe, but it's not going to change much and is nice to be able to see everything upfront! It'd be great to see this in private placeholders after the page kit migration though :D